### PR TITLE
[DOI] Adding JSON-LD description of LORIS in login page

### PR DESCRIPTION
**Pull request by xlecours**
This pull request is the result of FSCI Hackaton of August 1st 2017.

It get the json-ld (schema.org) description of a given DOI using the Datacite Content Resolver ( https://support.datacite.org/docs/datacite-content-resolver ). It look for a dc.Identifier meta tag in the document header to build the url that make an ajax call and add the result content to a script tag in the document header.

This allow for Search engines to know what the page is about and index it correctly. https://developers.google.com/search/docs/guides/sd-policies

In it current form it requires to add `data.datacite.org` in the connect-src of the CSPAdditionnalHeader config.

* Javascript code inspired from `https://search.datacite.org/javascripts/schemaorg.js`

Here is what should be added to the document.head
```
{
   "@context":"http://schema.org",
   "@type":"SoftwareApplication",
   "name":"LORIS",
   "applicationCategory":"Science",
   "applicationSubCategory":[
      "neuroimaging",
      "data management",
      "data querying",
      "imaging data",
      "behavioral data"
   ],
   "isBasedOn":"https://github.com/aces/Loris",
   "citation":{
      "@context":"http://schema.org",
      "@type":"ScholarlyArticle",
      "@id":"https://doi.org/10.3389/fninf.2011.00037",
      "url":"http://journal.frontiersin.org/article/10.3389/fninf.2011.00037/abstract",
      "additionalType":"JournalArticle",
      "name":"LORIS: a web-based data management system for multi-center studies",
      "author":[
         {
            "@type":"Person",
            "name":"Samir Das",
            "givenName":"Samir",
            "familyName":"Das"
         },
         {
            "@type":"Person",
            "name":"Alex P. Zijdenbos",
            "givenName":"Alex P.",
            "familyName":"Zijdenbos"
         },
         {
            "@type":"Person",
            "name":"Jonathan Harlap",
            "givenName":"Jonathan",
            "familyName":"Harlap"
         },
         {
            "@type":"Person",
            "name":"Dario Vins",
            "givenName":"Dario",
            "familyName":"Vins"
         },
         {
            "@type":"Person",
            "name":"Alan C. Evans",
            "givenName":"Alan C.",
            "familyName":"Evans"
         }
      ],
      "datePublished":"2012",
      "dateModified":"2015-02-18T19:21:41Z",
      "provider":{
         "@type":"Organization",
         "name":"Crossref"
      }
   },
   "operatingSystem":"GNU/Linux"
}
  ```

### CODE REVIEW

File: smarty/templates/public_layout.tpl [hunk 1]
```diff
@@ -6,12 +6,38 @@
...
 <head>
   <meta charset="UTF-8">
+  <meta name="DC.identifier" content="https://doi.org/10.3389/fninf.2011.00037">
   <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>{$page_title}</title>
   <link rel="stylesheet" href="{$baseurl}/bootstrap/css/bootstrap.min.css">
   <link rel="stylesheet" href="{$baseurl}/css/public_layout.css">
   <link type="image/x-icon" rel="icon" href="{$baseurl}/images/favicon.ico">
+  <script src="https://xavier-dev.loris.ca/js/jquery/jquery-1.11.0.min.js" type="text/javascript"></script>
```
Comment (driusan) [inline smarty/templates/public_layout.tpl]: The hostname is hardcoded here.

**Comment by ridz1208** [comment]
just adding needs work until we decide how this is going to move forward

the loris website: https://github.com/lorisdb/lorisdb.github.io

**Comment by ridz1208** [comment]
@xlecours do we need a meeting to discuss this one ? should we bring it up at a roadmap meeting ?

**Comment by xlecours** [comment]
@ridz1208 This should go on demo.loris.ca because of [that line](https://github.com/aces/Loris/pull/3024/files#diff-9f2e46b829c4833364f71c0fdfc44aedR9)

The DOI should come from the configs or database because it reference LORIS software itself. 

The discussion part would be :
1. Where manage DOI(s) of a study
2. Should there be a reference to LORIS software on each instance of LORIS.

**Comment by driusan** [comment]
This appears to be abandoned and should probably be on the loris.ca repo, not aces/Loris

**Pull request closed without merging**